docs/release-notes/webui-manage-templates.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**Improvements**
+
+-  Web UI: Allow user to manage templates through Web UI
```

Comment:
:orphan:

**Improvements**

-  WebUI: Allow users to create and manage configuration templates through the WebUI.

Reply:
@gt2345 i'll also create a docs PR for this to describe the steps

Reply:
That would be great, could you please stamp this PR so that it's ready for the release tomorrow